apis/utils.go

```diff
@@ -56,6 +56,15 @@ func Filter[T any](ts []T, filter func(T) bool) (filtered []T) {
 	return
 }
 
+func Find[T any](ts []T, predicate func(T) bool) (*T, bool) {
+	for _, v := range ts {
+		if predicate(v) {
+			return &v, true
+		}
+	}
+	return nil, false
```

Comment:
```suggestion
	for _, t := range ts {
		if predicate(t) {
			return &t, true
		}
	}
	return nil, false
```

pedantic but if its a range of ts then the value is a t 🤷‍♂️